src/en-GB/developers/index.html

```diff
@@ -28,7 +28,7 @@ <h2 class="h2 mb-12 lg:mb-16">Start using Ceph today</h2>
           software-defined storage system. Becoming a Ceph developer means you belong to a global community of passionate, talented and
           inspired developers, with the common goal of making Ceph the best it can be.
         </p>
-        <a class="button" href="/{{ locale }}/developers/documentation/">Get started with Ceph</a>
+        <a class="button" href="/{{ locale }}/users/documentation/">Get started with Ceph</a>
```

Comment:
Let's keep an eye on this one. I'm not sure the original intent was to do so much cross-linking between sections in the site (i.e. the "Getting started" page used to live here under Developers > Documentation). But let's see once we start to get more content through

Reply:
Agreed. As more content flows in we can adjust the cross-linking as necessary.